simulator-archetypes/archetype-mail/src/main/resources/archetype-resources/src/test/resources/logback-test.xml

```diff
@@ -22,7 +22,7 @@
     </encoder>
   </appender>
 
-  <logger name="com.consol.citrus" level="INFO"/>
```

Comment:
Should use both `com.consol.citrus` and `org.citrusframework` as the Citrus base project still uses `com.consol.citrus`